scripts/backup

```diff
@@ -39,7 +39,13 @@ ynh_backup --src_path="/etc/openvpn/client.conf" --not_mandatory
 ynh_backup --src_path="/etc/openvpn/client.cube" --not_mandatory
 ynh_backup --src_path="/etc/openvpn/client.ovpn" --not_mandatory
 
+ynh_backup --src_path="/etc/openvpn/scripts/route-up.d/10-set-firewall"
+ynh_backup --src_path="/etc/openvpn/scripts/route-up.d/20-set-dns"
+ynh_backup --src_path="/etc/openvpn/scripts/route-up.d/30-set-server-ipv6-route"
 ynh_backup --src_path="/etc/openvpn/scripts/route-up.d/40-set-ipv6"
+ynh_backup --src_path="/etc/openvpn/scripts/route-down.d/10-unset-firewall"
+ynh_backup --src_path="/etc/openvpn/scripts/route-down.d/20-unset-dns"
+ynh_backup --src_path="/etc/openvpn/scripts/route-down.d/30-unset-server-ipv6-route"
```

Comment:
hmm, don't we want to somehow backup/restore the entire folder ? to also include user-provided hook ? Or would that create funky stuff with hotspot maybe ?

Reply:
Yes exactly :/

As a workaround, we could follow some naming scheme, for instance XX-vpnclient-function and backup everything that contains vpnclient? or backup everything except when it matches another app name?

Reply:
@alexAubin What are your thoughts about this?

Reply:
No strong opinion, if the current code works let's go, if you think having the `-vpnclient-` intermediate prefix is better design, feel free to implement it :+1:

Reply:
I'll implement the prefix, it will be easier to switch later on.
Although for the hotspot, I didn't use a prefix, so we will need to change that :shrug:

Reply:
Hmm I don't know how I can restore the files… I can't restore the whole hook dirs because it will erase the hotspot script… and I can't restore each file separately because I was using the pattern `*-vpnclient-*` to backup them.